Content.Server/GameObjects/Components/Movement/AiControllerComponent.cs

```diff
@@ -60,13 +60,13 @@ public override void ExposeData(ObjectSerializer serializer)
         ///     Movement speed (m/s) that the entity walks.
         /// </summary>
         [ViewVariables(VVAccess.ReadWrite)]
-        public float WalkMoveSpeed { get; set; } = 4.0f;
+        public float CurrentWalkSpeed { get; set; } = 4.0f;
 
         /// <summary>
         ///     Movement speed (m/s) that the entity sprints.
         /// </summary>
         [ViewVariables(VVAccess.ReadWrite)]
-        public float SprintMoveSpeed { get; set; } = 10.0f;
+        public float CurrentSprintSpeed { get; set; } = 10.0f;
```

Comment:
Note: this is different to the Player one, is this intentional?

Reply:
Sounds like a case of "woops I didn't realize it's a duplicate value"